components/ModSubmissionsEditFormFacility.vue

```diff
@@ -353,7 +358,99 @@ function submitForm(e: Event) {
         return
     }
 
-    //submitthedatahere(googlemapsURL, mapLatitude, mapLongitude)
-    console.log('Form submitted!!! woohoo')
+    const updateSubmissionId = formSubmissionId || ''
+
+    if (!updateSubmissionId) {
+        console.error('Facility ID is required for updating the facility')
+        return
+    }
+
+    const facilityInputVariables = {
```

Comment:
this should definitely be a `type`

Reply:
yea you right. Unfortunately since it is a type with the property `id` I cannot use a more descriptive variable name such as `spoon`